SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -184,7 +184,7 @@ void USpatialReceiver::HandleActorAuthority(Worker_AuthorityChangeOp& Op)
 		}
 
 		// If we became authoritative over the position component. set our role to be ROLE_Authority
-		// and set our RemoteRole to be ROLE_AutonomousProxy iff the actor has an owning connection.
```

Comment:
Was this a type or was this supposed to mean "if and only if" (math style)?

Reply:
Perhaps, but is unnecessary and looks like a typo.

Reply:
Looks like it, lets make it explicit.